django/utils/log.py

```diff
@@ -90,24 +89,26 @@ def emit(self, record):
                  else 'EXTERNAL'),
                 record.getMessage()
             )
-            filter = get_exception_reporter_filter(request)
-            request_repr = '\n{}'.format(force_text(filter.get_request_repr(request)))
         except Exception:
             subject = '%s: %s' % (
                 record.levelname,
                 record.getMessage()
             )
             request = None
-            request_repr = "unavailable"
         subject = self.format_subject(subject)
 
+        # Create a copy of the record but omit information about the exception.
+        # We add nicely formatted traceback on our own.
+        no_exc_data = {k: v for k, v in record.__dict__.items() if k not in ('exc_info', 'exc_text')}
```

Comment:
Originally I hoped for something like `LogRecord(...)` but since it contains dynamically created attributes it wouldn't work. I modified the code as you suggested.